app/controllers/admin/dashboard_controller.rb

```diff
@@ -17,4 +18,10 @@ def valid_type?(type)
       :devices_detection]
     )
   end
+
+  def getVisits()
```

Comment:
Does the linter not pick this up? Can we add this to the rubocop config?

Reply:
Fixed, deleted this function